README.md

```diff
@@ -28,7 +28,7 @@ The following commands will perform a minimal installation:
 ```
 git clone https://github.com/ocaml-wasm/wasm_of_ocaml
 cd wasm_of_ocaml
-opam pin add dune.3.17 https://github.com/ocaml-wasm/dune.git#wasm_of_ocaml
+opam pin add https://github.com/ocaml-wasm/dune.git#wasm_of_ocaml
```

Comment:
What about this? This avoids installing all packages that comes with dune. And this pins dune with the right version.
```suggestion
opam pin add -n --with-version 3.17 https://github.com/ocaml-wasm/dune.git#wasm_of_ocaml
opam install dune
```

Reply:
This in fact seems required, to avoid https://github.com/ocaml-wasm/wasm_of_ocaml/issues/80